ros/launch/site.launch

```diff
@@ -24,6 +24,4 @@
     <!--Camera Info Publisher -->
     <include file="$(find camera_info_publisher)/launch/camera_info_publisher.launch"/>
 
-    <param name="sim_mode" value="0" type="int" />
```

Comment:
we do we delete it, how do we now can now theat ros launched from simulator or on site?

Reply:
Right now there is no code using "sim_mode" anywhere. The parameter is still defined in the params file though -- just in case we need it in some code when we run on Carla. The only parameter that we are using, that was similar to this is the "test_mode" parameter (which I've renamed to "cheat_mode" to make its purpose clearer). However, that parameter is not the same as "sim_mode", since we can have "cheat_mode = 0" even if "sim_mode = 1". Having such orphaned params sitting in our .launch files is just creating clutter for now.

Reply:
Yeah, I used it for a while, but now deleted, so resolve it